src/transport_ofi.c

```diff
@@ -663,19 +663,19 @@ int allocate_recv_cntr_mr(void)
 
 #if defined(ENABLE_MR_SCALABLE) && defined(ENABLE_REMOTE_VIRTUAL_ADDRESSING)
     ret = fi_mr_reg(shmem_transport_ofi_domainfd, 0, UINT64_MAX,
-                    FI_REMOTE_READ | FI_REMOTE_WRITE, 0, 0ULL, flags,
+                    FI_REMOTE_WRITE | FI_REMOTE_READ, 0, 0ULL, flags,
```

Comment:
Spurious change? I'm not the maintainer, so don't know if you guys care about changes like this.

Reply:
Thanks @sayantansur. This happened because of removing `FI_REMOTE_READ` at first and then adding it back. Will take care of this.